### ISSUE TITLE
Edit html table

### ISSUE THREAD

**Issue by andermikael** [RickStrahl/MarkdownMonster#932]
I'm not sure if this is suppose to work, but...

If I have an HTML table. The table is shown in the preview, but I want to make some changes to it.

I right click on the table tag and select Edit Table. The table is selected but then nothing!
I have also tried Format Table and got the same result...
Here is a table I tried with.

 ```html
<table>
  <tr>
    <th>Company</th>
    <th>Contact</th>
    <th>Country</th>
  </tr>
  <tr>
    <td>Alfreds Futterkiste</td>
    <td>Maria Anders</td>
    <td>Germany</td>
  </tr>
  <tr>
    <td>Centro comercial Moctezuma</td>
    <td>Francisco Chang</td>
    <td>Mexico</td>
  </tr>
</table> 
```
MM 2.4.6
Windows 10

// Anders

**Comment by RickStrahl** [comment]
Hmmm... not sure I have to check.

For now you can:

* Open the table editor and paste the HTML
* Edit and paste back

**Comment by RickStrahl** [comment]
Fixed for 2.4.6.3+
